src/fundus/publishers/us/rolling_stone.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class RollingStoneParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div.a-content > * > p.paragraph")
```

Comment:
The selector fails to parse pages like this:
https://www.rollingstone.com/music/music-lists/most-anticipated-coachella-sets-2024-1234999086/le-sserafim-1235000035/
https://www.rollingstone.com/music/music-lists/best-canadian-musicians-1234779020/

Reply:
Thanks, the lists have a slightly different structure than normal articles. The CSSSelector is fixed and works for both now
https://github.com/brandjakHU/fundus/blob/e8918df4fca60325ae08b4cfb56b09559be83efc/src/fundus/publishers/us/rolling_stone.py#L18

Reply:
Thanks for addressing this. Unfortunately, they still don't work for the articles I posted, further, the new selector selects a lot of empty paragraphs. I would suggest to use `div.a-content p.paragraph` which parses both correctly.

Reply:
Thanks, i changed it to `div.a-content p.paragraph`. I originally used only `p` because i found one list article which uses `<p>` nodes without the `paragraph`-class (https://www.rollingstone.com/music/music-lists/best-beyonce-songs-1378620/find-your-way-back-2019-1380991/). However, all other articles i looked at use `class=paragraph`, so maybe it is better to select like this and avoid empty paragraphs?